types/2020-08-27/index.d.ts

```diff
@@ -217,8 +217,7 @@ declare module 'stripe' {
     /**
      * API Errors
      */
-    static errors: Stripe.Errors;
```

Comment:
I guess we can't have this static anymore now that we have the `errors` namespace right?

Reply:
It's not that we can't it's that we don't need to. 

`static errors: typeof Stripe.errors` is equivalent to not having a static and `Stripe.errors` handled by namespace.

Reply:
Yep totally makes sense.